src/fides/api/models/connectionconfig.py

```diff
@@ -21,6 +22,9 @@
 from fides.api.schemas.saas.saas_config import SaaSConfig
 from fides.config import CONFIG
 
+if TYPE_CHECKING:
+    from fides.api.models.detection_discovery import MonitorConfig
+
```

Comment:
this is so mypy doesn't complain about not recognizing the `MonitorConfig` model. The `if TYPE_CHECKING` is needed to avoid a circular import